packages/bodiless-documentation/doc/Development/Guides/BuildingSites/README.md

```diff
@@ -1,111 +1,140 @@
+      - See: [CX Rich Text Editor Component : Site Builder Details](/Components/Editors/CX_Editors/CX_RTE_Editor#site-builder-details)
+    - `EditorPlain`: If your brand/site's font(s) require a superscript ® (registered trademark)
+      symbol, implement a version of the plain text editor which applies this CX token.
+      - See: [CX Plain Text Editor Component : Site Builder Details](/Components/Editors/CX_Editors/CX_PlainEditor#site-builder-details)
+
+01. FlowContainer/Layout Tool:
+    - Implement the basic layout editors for the site.
+      - `FlowContainer`: Apply standard spacing tokens to create one or more Flow Containers for
+        different use cases. Include your Rich Text editor.
+    - All the rest of the components will be added to the Flow Container as needed.
+
+01. Image Presets:
+    - Implement the basic image editors/presets for your site. In most cases, you can simply reuse
+      one or more of the default tokens available in CanvasX.
+
+01. Basic Layout / Page:
+    - Implement the site's basic page structure.
+    - Set up [responsiveness](./Responsiveness) (width of site/breakpoints, etc.).
+    - We recommend leaving most of the design elements as their CanvasX defaults for now.
+    - Utilize placeholders, keep styling limited to how the 'blocks' render in the component, and
+      ignore what renders.
+    - The end goal of the build will to be to replace brand-requirement working versions of:
+      - Logo
+      - Header and components in the Header
+      - Footer and components in the Footer
+      - Layout and components in the Layout
+
+01. Generic Page:
+    - Implement the site's generic page template.
+      - For components which have not yet been implemented, leave them at their defaults for now.
+    - `TitleRow`
+    - `GenericPage`
+      - Use appropriate Flow Containers for the content areas.
+      - Use the Layout from the previous step.
+    - `Page`
+      - Provide your `GenericPage` as the default variation in the `Page` chameleon.
+
+01. Organism tokens/components:
+    - Implement the basic organism customization tokens for the site.
+      - As you develop each, create a Flow Container token which makes relevant variations of the
+        organism available in one or more site Flow Containers, and add to existing Flow Containers.
+    - This list includes:
+      - [Accordion](/Components/SingleAccordion)
+      - [List/CompoundList](/Components/List)
+      - [Link](/Components/Link/)
+      - [YouTube](/Components/YouTube)
+      - Buttons
+    - If you're exploring an existing site for ideas, or performing a rebuild, review how to
+      [identify components and variations](./IdentifyingComponentsGuide) for some useful tips.
+
+01. Multi-language:
+    - If your site requires multi-language, we recommend bringing in the i18 CanvasX packages and
+      many components are depending on toggle data between languages.
```

Comment:
@hvanyo, this bullet needs some minor tweaking, but I was afraid to touch it and misinterpret your meaning.

Reply:
@misterjones  If your site require multi-language, we recommend I18 package..   TBD.